mobile/lib/ui/settings/TEMP/lock_screen_option_confirm_pin.dart

```diff
@@ -0,0 +1,386 @@
+import "package:flutter/material.dart";
+import "package:flutter/services.dart";
+import "package:photos/core/configuration.dart";
+import "package:photos/theme/ente_theme.dart";
+import "package:photos/ui/components/buttons/icon_button_widget.dart";
+import "package:pinput/pinput.dart";
+
+class LockScreenOptionConfirmPin extends StatefulWidget {
+  const LockScreenOptionConfirmPin({super.key, required this.pin});
+  final String pin;
+  @override
+  State<LockScreenOptionConfirmPin> createState() =>
+      _LockScreenOptionConfirmPinState();
+}
+
+class _LockScreenOptionConfirmPinState
+    extends State<LockScreenOptionConfirmPin> {
+  final _confirmPinController = TextEditingController(text: null);
+  final Configuration _configuration = Configuration.instance;
+  Key _pinputKey = UniqueKey();
```

Comment:
What is the use of this key? I haven't checked, just curious. Should be removed if it's unnecessary.